src/Mod/Path/PathScripts/post/gcode_pre.py

```diff
@@ -112,11 +112,9 @@ def insert(filename, docname):
 
         # Create a custom and viewobject
         obj = PathCustom.Create("Custom")
-        res = PathOpGui.CommandResources('Custom',
-            PathCustom.Create, PathCustomGui.TaskPanelOpPage,
-            'Path_Custom', 
-            QtCore.QT_TRANSLATE_NOOP('Path_Custom', 'Custom'), '', ''
-            )
+        res = PathOpGui.CommandResources('Custom', PathCustom.Create,
+                PathCustomGui.TaskPanelOpPage,
+                'Path_Custom', 'Custom', '', '')
```

Comment:
no more translate?

Reply:
It was throwing an error because QtCore wasn't imported.  I meant to get back and fix it proper but then got stuck.  These import scripts should be able to run headless so we need a better separation of GUI code here.
This is still importing PathOpGui and PathCustomGui.
Suggestions?

Reply:
`QtCore` is not the problem here though, it's not a UI class we should just import it.

`PathOpGui` and `PathCustomGui` are only needed for this part and this part should probably be in a
`if FreeCAD.GuiUp:`
clause - only need to set the `ViewObject` if there is a GUI ...

Reply:
Revised to add back the translation string.  Kicking the rest down the road